bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Decryption/DecryptionResult.cs

```diff
@@ -95,4 +97,17 @@ public DecryptionResult(string tallyId, bool isValid = true)
     {
         return self.SpoiledBallots;
     }
+
+    protected override void DisposeManaged()
+    {
+        base.DisposeManaged();
+        Tally?.Dispose();
+        ChallengedBallots?.ForEach(x => x.Dispose());
```

Comment:
We do not need a foreach here since we already have an extension for this for a list